fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java

```diff
@@ -286,7 +288,9 @@ public HeartbeatResponse call() {
                 // heartbeat to self
                 if (Catalog.getCurrentCatalog().isReady()) {
                     return new FrontendHbResponse(fe.getNodeName(), Config.query_port, Config.rpc_port,
-                            Catalog.getCurrentCatalog().getReplayedJournalId(), System.currentTimeMillis());
+                            Catalog.getCurrentCatalog().getReplayedJournalId(), System.currentTimeMillis(),
+                            Catalog.getCurrentCatalog().getFeStartTime(),
+                            Version.STARROCKS_VERSION + "-" + Version.STARROCKS_COMMIT_HASH);
```

Comment:
Does the commit hash is necessary?

Reply:
Yep.
1. Keep the display backends version information consistent。
2. When users use the master version to compile sr, they need to know which commit id was compiled based on.